tests/data/onboarding/index.js

```diff
@@ -1,25 +1,28 @@
+import genuine from "./genuine";
+import password from "./password";
+import analytics from "./analytics";
+
 export default {
   appTitle: "Ledger Live",
   duskColor: "#182532",
   darkColor: "#1c1d1f",
   lightColor: "#ffffff",
-  welcomeTitle: "Welcome to Ledger Live",
-  welcomeDesc:
-    "Let’s start by choosing how Ledger Live looks to you. You can change this later at any time in your settings.",
+  welcome: {
```

Comment:
Why are we testing against hardcoded data like this? I don't get it

Reply:
I agree. we should dynamically fetch the one from the en/app.json
we can literally import the JSON and use the json path to retrieve it.

Reply:
also to me, we should use it "in place". there is no need to have this tests/data folder
we must "collocate" the tests as much as possible to avoid looking at N different places.

Reply:
Actually there are different approaches: screenshots and text assertions

Given that screenshots are still not implemented, I'm using text assertions.

About the hard coded data, it's a topic that we should discuss on:
- Different languages support (i18n?)
- Is it relevant to use production data for expected test data ?
- Easiest way to maintain

Which strategy ?
100% Visual regression testing or a mix...